frontend/src/routing/associations.ts

```diff
@@ -9,13 +9,35 @@ import { AssociationFilesystemDetail } from "../pages/associations/files/Detail"
 import { AssociationFilesystemEdit } from "../pages/associations/files/Edit";
 import { AssociationFilesystemUpload } from "../pages/associations/files/Upload";
 
+// To be replaced
+import { AssociationListEvent } from "../pages/associations/events/List";
+import { AssociationCreateEvent, AssociationEditEvent } from "../pages/associations/events/Edit";
+
 export const routes = association => [
     {
         path: `/`,
         component: AssociationHome,
         exact: true,
         props: { association: association }
     },
+    {
+        path: `/events/`,
+        component: AssociationListEvent,
```

Comment:
Switch "List" and "Event" to always go from the most general to the most specific. Valid for the next components